drivers/focuser/lacerta_mfoc.cpp

```diff
@@ -474,15 +476,18 @@ uint32_t lacerta_mfoc::GetAbsFocuserPosition()
 
     int nbytes_written = 0;
     int nbytes_read = 0;
+    int count = 0;
+
+    tty_write_string(PortFD, MFOC_cmd, &nbytes_written);
+    LOGF_INFO("CMD <%s>", MFOC_cmd);
```

Comment:
LOGF_DEBUG should be used. LOGF_INFO is for informational messages, not debug.

Reply:
changed to debug